test/tests/functional/pbs_test_tpp.py

```diff
@@ -81,11 +109,100 @@ def submit_resv(self, set_attrib=None, exp_attrib=None):
+            self.server.expect(JOB, 'queue', id=jid, op=UNSET, offset=10)
+            self.server.log_match("%s;Exit_status=0" % jid)
+            # Submit multi-chunk job
+            set_attrib = {'Resource_List.select': '2:ncpus=1',
+                          ATTR_l + '.place': 'scatter', ATTR_k: 'oe'}
+            jid = self.submit_job(set_attrib, exp_attrib={'job_state': 'R'},
+                                  job_script=True)
+            self.server.expect(JOB, 'queue', id=jid, op=UNSET, offset=10)
+            self.server.log_match("%s;Exit_status=0" % jid)
+        if interactive_job:
+            # Submit interactive job
+            set_attrib = {'Resource_List.select': '2:ncpus=1',
+                          ATTR_inter: '',
+                          ATTR_l + '.place': 'scatter', ATTR_k: 'oe'}
+            jid = self.submit_job(set_attrib, exp_attrib={'job_state': 'R'},
+                                  interactive=True)
+            self.server.expect(JOB, 'queue', id=jid, op=UNSET)
+        if resv:
+            # Submit reservation
+            set_attrib = {'Resource_List.select': '2:ncpus=1',
+                          ATTR_l + '.place': 'scatter',
+                          'reserve_start': int(time.time() + 10),
+                          'reserve_end': int(time.time() + 120)}
+            exp_attrib = {'reserve_state': (MATCH_RE, 'RESV_CONFIRMED|2')}
+            rid = self.submit_resv(set_attrib, exp_attrib)
+            resv_que = rid.split('.')[0]
+            # Submit job into reservation
+            set_attrib = {'Resource_List.select': '2:ncpus=1',
+                          ATTR_q: resv_que, ATTR_k: 'oe',
+                          ATTR_l + '.place': 'scatter'}
+            jid = self.submit_job(set_attrib, job_script=True)
+            # Wait for reservation to start
+            a = {'reserve_state': (MATCH_RE, 'RESV_RUNNING|5')}
+            self.server.expect(RESV, a, rid, offset=10)
+            self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+
+    def pbs_restart(self, host_name):
+        """
+        This function starts PBS daemons after updating pbs.conf file
+        :param host_name: Name of the host on which PBS
+                          has to be restarted
+        :type host_name: String
+        """
+        pi = PBSInitServices(hostname=host_name)
+        pi.restart()
+
+    def set_conf(self, host_name, attribs):
+        """
+        This function sets attributes in pbs.conf file
+        :param host_name: Name of the host on which pbs.conf
+                          has to be updated
+        :type host_name: String
+        :param attribs: Attributes to be updated in pbs.conf
+        :type attribs: Dictionary
+        """
+        pbsconfpath = self.du.get_pbs_conf_file(hostname=host_name)
+        self.du.set_pbs_config(hostname=host_name, fin=pbsconfpath,
+                               confs=attribs)
+        self.pbs_restart(host_name)
+
+    def unset_pbs_conf(self, host_name):
```

Comment:
let us make unset_pbs_conf() a counterpart to set_pbs_conf() and accept attribs.